pkgs/desktops/gnome-3/misc/libdazzle/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, pkgs, fetchFromGitHub, ninja, meson, pkgconfig, glib, gtk3, vala, ...}:
+
+let
+  version = "3.27.2";
+in
+stdenv.mkDerivation {
+  name = "libdazzle-${version}";
+
+  buildInputs = [ glib gtk3 ];
+  nativeBuildInputs = [ ninja meson pkgconfig vala ];
+
+  enableParallelBuilding = true;
```

Comment:
Derivations using meson are built in parallel by default: #32271

Reply:
Thanks, removed.